[DOCSP-41529] cleanup ticket and organization (#65)
mongodb/docs-java-rs: source/write-data-to-mongo.txt

Be sure to replace all placeholders in the code examples, such as ``<connection string>``, with
the relevant values for your MongoDB deployment.

This guide uses the {+pr+} library to consume ``Publisher`` instances returned
by the {+driver-short+} methods. To learn more about the {+pr+} library
and how to use it, see `Getting Started <https://projectreactor.io/docs/core/release/reference/#getting-started>`__
in the Reactor documentation. 

> There are also other ways to consume ``Publisher`` instances. You can use one of many alternative libraries such as
> `RxJava <https://github.com/ReactiveX/RxJava>`__ to consume ``Publisher``
> instances, or consume ``Publisher`` instances by calling ``Publisher.subscribe()`` directly and passing your own

Reviewer: S: to avoid the repetition of "to consume ``Publisher`` instances":

```suggestion
Alternatively, you can consume ``Publisher``
instances by using an external library such as
`RxJava <https://github.com/ReactiveX/RxJava>`__  or by calling ``Publisher.subscribe()`` directly and passing your own
```
